app/models/concerns/document_export_behavior.rb

```diff
@@ -90,5 +90,7 @@ def download_title
 
   def generator_url
     report_class.url
+  rescue NoMethodError
```

Comment:
Update `app/views/account_downloads/_list.haml` (which seems to be the only user of this method) to have a `link_to_if` rather than editing bad URLs?

Reply:
It was throwing an error because report_class.url was being called on `Class`.  Changing to `report_class&.url` didn't fix that since report_class wasn't nil.  How would you catch that in the list?

Reply:
I was just thinking that we could just not generate the link if `generator_url` returns nil... And since the text is just 'Report', I think just leave it out...
```
- if item.generator_url.present?
  %td= link_to 'Report', "#{item.generator_url}?#{item.query_string}"
- else
  %td
```
?